cli/provider/cmd.go

```diff
@@ -737,6 +740,11 @@ func (c *CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 		}
 	case "record", "test", "rerecord":
 
+		if cmd.Name() == "rerecord" {
+			updateTestSet, _ := cmd.Flags().GetBool("amend-testset")
```

Comment:
Amend-testset is setting the variable CreateTestSet in the config. Why can't we just use the same names for better understanding?